aiida/common/datastructures.py

```diff
@@ -104,6 +104,14 @@ class CodeInfo(DefaultFieldsAttributeDict):
 
         code.x cmdline_params[0] cmdline_params[1] ... < stdin > stdout
 
+    * ``folder_path``: (optional) the path, in string, of the folder where to
```

Comment:
I would opt for `run_directory` or `relative_run_directory`. The first one is clear I think, but it might give the wrong impression that it can be used to control the absolute run path which is not the case. Adding `relative_` as a suffix makes it crystal clear that this is a relative path, within the automatically generated working directory for the job.

Reply:
I do agree very much with the second one. Very clear indeed.